README.md

```diff
@@ -22,7 +22,7 @@ DataDog::DogStatsd - A Perl client for DogStatsd
 
 Statsd: A DogStatsd client (https://www.datadoghq.com)
 
-The difference between [Net::Statsd](https://metacpan.org/pod/Net::Statsd) and this is that it supports tags and namespace.
+The difference between [Net::Statsd](https://metacpan.org/pod/Net%3A%3AStatsd) and this is that it supports tags and namespace.
```

Comment:
ok... but why though?

Reply:
not done by me. This happened during `dzil run cpanm -l /home/git/regentmarkets/cpan/local .`